test/e2e/storage/external/external.go

```diff
@@ -422,5 +422,10 @@ func (d *driverDefinition) PrepareTest(ctx context.Context, f *framework.Framewo
 		Framework:           f,
 		ClientNodeSelection: e2epod.NodeSelection{Name: d.ClientNodeName},
 	}
+
+	if framework.NodeOSDistroIs("windows") {
+		e2econfig.ClientNodeSelection.Selector = map[string]string{"kubernetes.io/os": "windows"}
+	}
```

Comment:
l.config.ClientNodeSelection.Selector is specified always as `Selector` in the `podConfig`.
Instead of empty selector of linux, how about setting `linux` like
```suggestion
	} else {
		e2econfig.ClientNodeSelection.Selector = map[string]string{"kubernetes.io/os": "linux"}
	}
```
?

Reply:
Thanks for the suggestion 👍